src/option.cpp

```diff
@@ -2297,10 +2296,6 @@ void process_option_line(char *configLine, const char *filename)
          add_keyword(args[idx], CT_TYPE);
       }
    }
-   else if (strcasecmp(args[0], "define") == 0)
```

Comment:
Hmm...we *could* have this omit a more specific warning (and get rid of it fully in a few releases)... up to you though.